### DISCUSSION TITLE
Stuck on step 5

### DISCUSSION THREAD

**Discussion by off-wyt** [skills/.github#342] [Communicate using Markdown]
I've merged my pull request and waited >20 seconds for GH actions to take me to the next step, but I'm stuck. How can i figure out what to do to move on?

**Comment by cmwilson21** [comment]
Are you still stuck on this course?

**Reply by Antismart** [reply]
I'm also stuck on step 1 on the part of editing the index.md file. i can't edit the index.md file

**Reply by sinsukehlab** [reply]
@Antismart I'm afraid that you haven't started the course.

**Reply by TuckWai97** [reply]
@Antismart 
I found out how to do this after 3 attempts.
After you press "Create pull request" green button at the place of "Compare changes", you need to click "Create pull request" again. 
Then you will be in the tab of "Conversation", check this below your pull request title, mine is "**Start markdown #2"**.
The "File changed" tab is the last tab, you need to switch to that tab, and edit the index.md file.
Hope this could help 😀

**Reply by Antismart** [reply]
thanks 😊 it's working
